miniapp/USERGUIDE.md

```diff
@@ -264,44 +264,57 @@ val miniAppMessageBridge = object: MiniAppMessageBridge() {
 }
 
 val userInfoBridgeDispatcher = object : UserInfoBridgeDispatcher() {
-    override fun getUserName(): String {
-        var name: String = ""
-        // Implementation details to get user name
+
+    override fun getUserName(
+        onSuccess: (userName: String) -> Unit,
+        onError: (message: String) -> Unit
+    ) {
+        val name: String = ""
+        // Check if there is any valid username in HostApp
         // .. .. ..
-        return name
+        if (name is valid)
```

Comment:
It seems like this could be confusing to developers because `is` is a valid operator in Kotlin but it wouldn't be used in a context like this. So the code would be broken if they just copied and pasted it like this. Maybe we can just make `name` be nullable here and add a comment?
```
val name: String?
 // Check if there is any valid username in HostApp
// .. .. ..
if (name) // Check if name is valid
```

Reply:
Ah right, I didn't notice like this way. I have updated this part.